roles/openshift_control_plane/tasks/static_shim.yml

```diff
@@ -10,10 +10,10 @@
   - "scripts/{{ l_runtime }}/master-logs"
   - "scripts/{{ l_runtime }}/master-restart"
   vars:
-    l_runtime: "{{ 'crio' if openshift_use_crio | default(False) else 'docker' }}"
+    l_runtime: "{{ 'crio' if (openshift_use_crio | default(False) | bool) else 'docker' }}"
```

Comment:
Yes, now that you mention it, maybe openshift_use_crio belongs in openshift_facts/defaults/main.yml as well, so we don't drift.